src/content/docs/guides/networking/2-restful-api-call.mdx

```diff
@@ -2,8 +2,8 @@
 title: How to make a RESTful API call using SplashKit
 description: In the world of web services and microservices many companies make their data availlable through a REST APIIn this document we want to use SplashKit to make a RESTful API call.
```

Comment:
Correct the spelling of "available," add a space between "API" and "In," and insert a comma after "microservices" for clarity.

Reply:
Just fixed this